bom/pom.xml

```diff
@@ -299,4 +299,53 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <!-- dependency on code style module -->
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-code-style</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+
+      <!-- enable checkstyle for all inherited modules -->
+
+      <plugin>
```

Comment:
this whole block should go into the parent POM FMPOV

Reply:
Same here, this can't go into the parent POM as described above.